app/config/sql/board2.sql

```diff
@@ -0,0 +1,48 @@
+--
+-- Create database
+--
+CREATE DATABASE IF NOT EXISTS board;
+GRANT SELECT, INSERT, UPDATE, DELETE ON board.* TO board_root@localhost
+IDENTIFIED BY 'board_root';
+FLUSH PRIVILEGES;
+
+USE board;
+
+--
+-- Restart from scratch.
+--
+
+DROP TABLE IF EXISTS comment;
+DROP TABLE IF EXISTS user;
+DROP TABLE IF EXISTS thread;
+
+--
+-- Create tables
+--
+
+CREATE TABLE IF NOT EXISTS thread (
+    id        INT UNSIGNED NOT NULL AUTO_INCREMENT,
+    title     VARCHAR(255),
```

Comment:
Does this mean that you're accepting titles up to 255 characters po?
If not, you could minimize the required size for title here po since you already have a validation for the title length :bow: